lib/datadog/tracing/contrib/aws/services/kinesis.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative '../ext'
+
+def add_kinesis_tags(span, params)
```

Comment:
As a side-point, immaterial after the proposed changes: a method declaration like this, at the top of the file, not inside any `class` or `module` creates a global function in the current process.
This means, for example, `add_kinesis_tags` is globally available for the client to invoke, from pretty much anywhere in their code base. This not desirable for a library like ours to do because it pollutes the application naming space.
In the worst case scenario, the user has a `add_kinesis_tags` method already and we just overwrote it here.